presto-spark-base/src/main/java/com/facebook/presto/spark/execution/nativeprocess/NativeExecutionModule.java

```diff
@@ -74,17 +72,20 @@ protected void bindShuffle(Binder binder)
 
     protected void bindWorkerProperties(Binder binder)
     {
-        newOptionalBinder(binder, new TypeLiteral<WorkerProperty<?, ?, ?>>() {
-        }).setDefault().to(PrestoSparkWorkerProperty.class).in(Scopes.SINGLETON);
-        if (connectorConfig.isPresent()) {
-            binder.bind(PrestoSparkWorkerProperty.class).toInstance(
-                    new PrestoSparkWorkerProperty(connectorConfig.get(),
-                            new NativeExecutionNodeConfig(), new NativeExecutionSystemConfig(
-                            ImmutableMap.of())));
+        // Bind NativeExecutionCatalogProperties - this is not bound elsewhere
+        if (catalogProperties.isPresent()) {
```

Comment:
Given the `NativeExecutionCatalogProperties` is accommodative to null input, we probably don't need this if-else here. 
Probably just change this to 
```
new NativeExecutionCatalogProperties(catalogProperties.orElse(ImmutableMap.of()))
```